src/utils/constants.ts

```diff
@@ -0,0 +1 @@
+export const DEFAULT_NAMESPACE = 'default';
```

Comment:
How is this used?

We should avoid putting anything in the `default` namespace